crawl.js

```diff
@@ -232,6 +232,59 @@ async function updateLatestSignedBlock () {
     }
 }
 
+async function getPastEvent () {
+    let blockNumber = await web3.eth.getBlockNumber()
+    console.log('Get all past event from block 0 to block', blockNumber)
+    validator.getPastEvents('allEvents', { fromBlock: 0, toBlock: blockNumber }, async function (error, events) {
```

Comment:
fromBlock should not start from 0, should start from biggest block number that get from DB.
If db is empty, start from 0